spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe InvenioRdmRecordConverter do
+  let(:user) { FactoryGirl.create(:user) }
+  let(:generic_file) { make_generic_file(user, doi: ["doi:123/ABC"]) }
+  let(:json) { "{\"pids\":{\"doi\":{\"identifier\":\"#{generic_file.doi.shift}\",\"provider\":\"datacite\",\"client\":\"digitalhub\"}},\"provenance\":\"#{user.username}\"}" }
```

Comment:
As we add new fields to `InvenioRdmRecordConverter`, we can just add our expected values to this.

Reply:
Sounds good to me. Is provenance marked with # because we're not sure what to put in this field currently? I'm going to follow your lead here after you merge to make my own specs and I think that would be a decent standard for fields we don't know how to deal with right now. Similarly, for ambiguous mappings I think we should have some sort of standard for how we delimit things. Right now i'm using semi-colons to separate, but I think i'll switch it to commas so it's more in line with csv. You will see what i'm talking about later when I open that PR, but it's kind of an odd situation.

Reply:
Oh, i just added provenance here as it was in `InvenioRdmRecordConverter`. I did not realize it was not what we want, but I can address that in #785

Reply:
Oh, I see. I sort of forgot what was there, but yeah I think for now we can leave the # and just address it when the time comes.